Dockerfile-ubuntu1804

```diff
@@ -6,22 +6,22 @@ CMD          bash
 RUN apt-get update && apt-get -y install \
   apt-utils \
   build-essential \
+  clang-9 \
```

Comment:
This is just organizing the packages in alphabetical order, right?
Did you use a docker linter to order them?

Reply:
Yes it is, I use a "sort lines" plugin + docker format in my Visual Code editor.